password_security/__openerp__.py

```diff
@@ -6,6 +6,7 @@
     'name': 'Password Security',
     "summary": "Allow admin to set password security requirements.",
     'version': '8.0.1.1.0',
```

Comment:
Looks like the conflict resolution failed

Reply:
Thanks for noting! Fixed.